Packs/GitHub/Integrations/GitHub/GitHub.py

```diff
@@ -1611,6 +1611,34 @@ def get_github_get_check_run():
     return_results(command_results)
 
 
+def create_release_command():
+    args = demisto.args()
+    tag_name = args.get('tag_name')
+    data = {
+        'tag_name': tag_name,
+        'name': args.get('name'),
+        'body': args.get('body'),
+        'draft': argToBoolean(args.get('draft'))
```

Comment:
[PEP8](https://www.python.org/dev/peps/pep-0008/#when-to-use-trailing-commas)
```suggestion
        'draft': argToBoolean(args.get('draft')),
```